### PR TITLE
Feature maUI #70

### PR DESCRIPTION
**Pull request by PajakKonrad**

### CODE REVIEW

File: MainPage.xaml [hunk 1]
```diff
@@ -20,20 +20,31 @@
             <!-- Add record -->
             <Frame CornerRadius="10" Padding="20" BackgroundColor="#373737">
                 <VerticalStackLayout Spacing="15">
+                    <Picker x:Name="MediaTypePicker">
```
Comment (byVirrox) [inline MainPage.xaml:23]: Hardcoded Picker

File: MainPage.xaml.cs [hunk 2]
```diff
@@ -161,6 +172,102 @@ private void OnFilterRecord(object sender, EventArgs e)
...
+            FilterMediaAgencies("TV", "TVValue");
+        }
+
+        private void OnPressFilterButtonClicked(object sender, EventArgs e)
+        {
+            // Call the FilterMediaAgencies method for "Press" with a specific filter value
+            FilterMediaAgencies("Press", "PressValue");
+        }
+
+        private void OnAddMediaRecord(object sender, EventArgs e)
```
Comment (byVirrox) [inline MainPage.xaml.cs:214]: maybe split the method into smaller parts

File: MainPage.xaml (hunk 1 above)
Comment (byVirrox) [inline MainPage.xaml:23]: Hardcoded Picker

File: MainPage.xaml.cs [hunk 3]
```diff
@@ -243,6 +353,39 @@ private void OnDeleteRecord(object sender, EventArgs e)
                 }
             }
         }
+        /// <summary>
+        /// Method to copy all stored data to the clipboard for a given table
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
```
Comment (byVirrox) [inline MainPage.xaml.cs:360]: good

**Review by byVirrox — COMMENTED** [review]
Everything good so far. The filters are not quite working yet

File: MainPage.xaml (hunk 1 above)
Comment (PajakKonrad) [reply MainPage.xaml:23]: How can you avoid hardcoding the objects ¬¬?

File: MainPage.xaml.cs (hunk 2 above)
Comment (PajakKonrad) [reply MainPage.xaml.cs:214]: will do

**Pull request closed without merging**